src/components/typography/typography-v2.ts

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+import { css } from 'styled-components';
+
+const timestamp = css`
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 110%;
+  letter-spacing: -0.01em;
+
+  color: #000000;
+`;
+
+const toplineR = css`
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 110%;
+
+  color: #000000;
+`;
+
+const toplineS = css`
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 110%;
+
+  color: #000000;
+`;
+
+const headlineXL = css`
+  font-weight: bold;
+  font-size: 48px;
+  line-height: 110%;
+
+  letter-spacing: -0.01em;
+
+  color: #000000;
+`;
+
+const headlineL = css`
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 110%;
+
+  letter-spacing: -0.01em;
+
+  color: #000000;
+`;
+
+const headlineM = css`
+  font-weight: bold;
+  font-size: 28px;
+  line-height: 110%;
+
+  letter-spacing: -0.01em;
+
+  color: #000000;
+`;
+
+const headlineS = css`
+  font-weight: bold;
+  font-size: 20px;
+  line-height: 110%;
+
+  letter-spacing: -0.01em;
+
+  color: #000000;
+`;
+
+const headlineXS = css`
+  font-weight: bold;
+  font-size: 16px;
+  line-height: 150%;
+
+  letter-spacing: -0.01em;
+
+  color: #000000;
+`;
+
+const textL = css`
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 150%;
+
+  letter-spacing: -0.01em;
+
+  color: #000000;
+`;
+
+const textR = css`
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 150%;
+
+  letter-spacing: -0.01em;
+
+  color: #000000;
+`;
+
+const textS = css`
+  font-weight: normal;
+  font-size: 14px;
+  line-height: 150%;
+
+  letter-spacing: -0.01em;
+
+  color: #000000;
+`;
+
+const textXS = css`
+  font-weight: normal;
+  font-size: 12px;
+  line-height: 150%;
+
+  letter-spacing: -0.01em;
+
+  color: #000000;
+`;
+
+const quote = css`
+  font-style: italic;
+  font-weight: normal;
+  font-size: 20px;
+  line-height: 150%;
+
+  color: #000000;
+`;
+
+const code = css`
+  font-family: Monaco;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 150%;
+
+  color: #000000;
+`;
+
+const menuItemR = css`
+  font-weight: bold;
+  font-size: 48px;
+  line-height: 150%;
+
+  letter-spacing: -0.01em;
+
+  color: #000000;
+`;
+
+const menuItemS = css`
+  font-weight: bold;
+  font-size: 32px;
+  line-height: 150%;
+
+  letter-spacing: -0.01em;
+
+  color: #000000;
+`;
+
+// MenuLaguage has the exact same configuration as MenuMeta
+const menuMeta = css`
+  font-weight: bold;
+  font-size: 14px;
+  line-height: 110%;
+
+  color: #000000;
+`;
+
+export const TextStyles = {
+  timestamp,
+  toplineR,
+  toplineS,
+  headlineXL,
+  headlineL,
+  headlineM,
+  headlineS,
+  headlineXS,
+  textL,
+  textR,
+  textS,
+  textXS,
+  quote,
+  code,
+  menuItemR,
+  menuItemS,
+  menuMeta,
+};
```

Comment:
Yep, was also thinking about this. Maybe inlining all those objects can become a bit messier (for example when you want to use some extra constants or so), but right now it would be more simple than the separate objects.

Reply:
Yes, I do agree, I implemented the changes. Please have another look at the changes.

Reply:
Thanks for the feedback :pray: